.github/workflows/php-security.yml

```diff
@@ -0,0 +1,47 @@
+name: PHP Security analysis
+
+on:
+  push:
+    branches: [ main ]
+  pull_request:
+    branches: [ main ]
+  schedule:
+    - cron: '0 0 * * 1'
+
+jobs:
+  snyk:
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        stability: [prefer-lowest, prefer-stable]
```

Comment:
Is it also worth adding another matrix param for PHP versions? 8.0, 8.1 and 8.2? I know we do so for tests, but wondering if we need for security and static too?